emails.py

```diff
@@ -63,14 +25,37 @@ def emailExtractor(urlString):
     return emailList
 
 
+def getMPDetails(postcode):
+    url_base = "http://api.postcodes.io/postcodes/"
+    with urllib.request.urlopen(url_base + postcode) as url:
+        data = json.loads(url.read().decode())
+        if data["status"] == 200:
+            # Create query from the data retrieved from postcodes.io
+            query = {"constituency": data["result"]["parliamentary_constituency"]}
+        else:
+            raise KeyError("No postcode found!")
+
+    # Tell Python where to look for the database.
+    client = pymongo.MongoClient(
+        "mongodb://heroku_b22mk7d6:mpdj7v335osvtda7c3g3ffo2ao@ds121565.mlab.com:21565/heroku_b22mk7d6"
+    )
+    # Define where the data is stored.
+    mpCollection = client["heroku_b22mk7d6"]["mp_email_list"]
+
+    # Execute the query on the mpCollection
+    mpDetails = mpCollection.find_one(query)
+    # Return a dictionary of the email, name and constituency
+    return mpDetails
+
+
 def draftEmails(myname, postcode):
-    ret = getGovDetails(postcode)
-    ward = ret["ward"]
```

Comment:
The reason the ward is returned here is to fill a template that requires it, I really like the MongoDB integration (and it looks solid & is well documented - thank you!) but would request that in line 31 where data is gathered from the postcodes api, we store the ward and return that with the MP information too.